test.sh

```diff
@@ -1,4 +1,19 @@
 #!/usr/bin/env bash
-set -e
+set -ex
 
-./_integration-test/run.sh
\ No newline at end of file
+echo "Ensure customizations work"
+cat <<EOT > sentry/enhance-image.sh
+#!/bin/bash
+touch /created-by-enhance-image
+apt-get update
+apt-get install -y gcc libsasl2-dev python-dev libldap2-dev libssl-dev
+EOT
+printf "python-ldap" > sentry/requirements.txt
+
+echo "Testing initial install"
+./install.sh
+./_integration-test/run.sh
+
+echo "Testing in-place upgrade"
```

Comment:
What do you think about moving writing commands into `enhance-image.sh` in second run with `--minimize-downtime`?

Reply:
Good idea, I will extract the validation for it and add a before-after comparison to ensure the test itself is sound.